### ISSUE TITLE
Not playing preferred sound

### ISSUE THREAD

**Issue by LocalBoomer** [NoiSek/Sound-Alert#3]
So I've tested it again with the newest version and now everything 'works'

however the only file in the directory (sound.wav) isn't played. Instead it keeps playing the (I assume default) error sound from windows (as in the one you get for trying to click on a disabled dialog box)

Any ideas why this is?

P.S. I have used the get command and it says it is using the correct directory.

**Comment by NoiSek** [comment]
What directory do you have it set to?

**Comment by LocalBoomer** [comment]
C:\Users<me>\AppData\Roaming\HexChat\sounds

Edit 1: I just tried another folder and same issue.

**Comment by NoiSek** [comment]
Can only guess this is either a permissions issue or something to do with subdirectories: 68d4b11c9761fd7c29b086707154eb10b785d9ef

You sure have managed to find a lot of edge cases. Just goes to show this is why you write Unit Tests.

**Comment by LocalBoomer** [comment]
So I tried the newest update and it now works off of my backup hard disk. still no luck using it off my OS drive so I'm pretty sure you're right about the permissions thing. Any idea how to fix or circumvent this? (But hey it works! :D)

Ok so now it works properly turns out I just needed to reload the plugin after restarting hexchat (shouldn't be much of an issue as I don't plan to change the sound often)

Thanks so much for all your time listening to me breaking your plugin! \o/
